### DISCUSSION TITLE
Product Filter

### DISCUSSION THREAD

**Discussion by kofitones** [inspireui/support#5692] [New Feature/Ideas Requests] [answered]
Can we get a better product filtering for the fluxstore woocommerce ?  Currently all attributed show on every category selected and this doesn't provide a good user experience. It makes search for its extremly difficult on the app.

**Comment by HanielQC** [comment, marked as answer]
It is better with flexible options now (FluxStore 3.8.0) - The sale 30% OFF this March 2023 happens.
Thank you
